.github/workflows/ci-sage.yml

```diff
@@ -0,0 +1,155 @@
+name: Run Sage CI for Linux
+
+## This GitHub Actions workflow provides:
+##
+##  - portability testing, by building and testing this project on many platforms
+##
+##  - continuous integration, by building and testing other software
+##    that depends on this project.
+##
+## It runs on every pull request and push of a tag to the GitHub repository.
+##
+## The testing can be monitored in the "Actions" tab of the GitHub repository.
+##
+## After all jobs have finished (or are canceled) and a short delay,
+## tar files of all logs are made available as "build artifacts".
+##
+## This GitHub Actions workflow uses the portability testing framework
+## of SageMath (https://www.sagemath.org/).  For more information, see
+## https://doc.sagemath.org/html/en/developer/portability_testing.html
+
+## The workflow consists of two jobs:
```

Comment:
I think it may be better to add these jobs to the existing workflow making it possible to integrate all the jobs in a single branch protection check job depending on the required jobs.

Reply:
That could also be done via https://docs.github.com/en/actions/using-workflows/reusing-workflows

Reply:
I have a preference for `ci-sage.yml` to be a separate file because I have provided versions of it for various projects, so it's easier for me to keep it up to date.

Reply:
Yep, reusable workflows are a good idea except they don't work well within the same repository (because you cannot refer to the same commit when including them).

Reply:
I didn't know that, thanks

Reply:
I don't have access to this repo like Jason does :) But I still occasionally review things to help out.

As a recommendation to you personally — you could create a repository dedicated to this reusable workflow and parameterize it. This way you could contribute it as reusable to many projects but only the list of the matrix targets would need to be maintained on this side. Then, you'd occasionally update the SHA-1 or the tag in repos where contribute, instead of having to synchronize the whole file.

Reply:
Thanks, that's a good idea. Noted at https://trac.sagemath.org/ticket/29060

Reply:
I slightly prefer the separate workflow because it provides separation based on custody/origin. Good suggestions on limiting the amount of copy/synchronization. Since this project doesn't use protections, let's go with this proposal for now and consider merging separately.